buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    `kotlin-dsl`
```

Comment:
why is this needed?

Reply:
this makes it possible to use the gradle.kts scripts in the buildSrc as actual plugins